VTUL/dlp-access
# Fixed typo in fullwidth class, added css classes for search bar form elements

**The title of this pull-request should be a brief description of what the pull-request fixes/improves/changes. Ideally 50 characters or less.**
* * *
Fixed typo in fullwidth class, added css classes for search bar form elements

**JIRA Ticket**: (link) (:star:)
https://webapps.es.vt.edu/jira/browse/LIBTD-1987

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Fixed typo in class name "fullwidth" use in ItemsPage.js and SearchPage.js. Also added existing css classes to SearchBar.js so that search bar is correct width, responsive to page size, and button style matches search bar a bit better.

# What's the changes? (:star:)
A in-depth description of the changes made by this PR. Technical details and possible side effects.
Fixed typo in class name "fullwidth" use in ItemsPage.js and SearchPage.js. Also added existing css classes to SearchBar.js so that search bar is correct width, responsive to page size, and button style matches search bar a bit better.

Example:
* Changes x feature to such that y
* Added new feature x
* Removed y to fix z

# How should this be tested?
Search bar on the "Browse items" page should have the input field and search button on the same line, and should resize in response to the page. It should look this way both before and after pressing the "GO" button.

A description of what steps someone could take to:
Check that the elements are on the same line both before and after a search and at different screen sizes.

# Additional Notes:
Branch is LIBTD-1987
The classes used were all pre-existing in the root.css file.

Example:
* What branch to be used for testing this PR?
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository? 
* Could this change impact execution of existing code?

# Interested parties
Tag (@ mention) interested parties
@whunter 

(:star:) Required fields
